recipe/meta.yaml

```diff
@@ -1,47 +1,58 @@
 {% set name = "frozendict" %}
-{% set version = "2.3.4" %}
-{% set sha256 = "15b4b18346259392b0d27598f240e9390fafbff882137a9c48a1e0104fb17f78" %}
+{% set version = "2.4.2" %}
+{% set sha256 = "741779e1d1a2e6bb2c623f78423bd5d14aad35dc0c57e6ccc89e54eaab5f1b8a" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
I think `pypi.io` is a preferred domain in the `aggregate` repo:
```
╰─❯ rg "url: https://pypi.io" | wc
    1390   12952  183252
╰─❯ rg "url: https://pypi.org" | wc
      90     730   10747
```
Were there any troubles with `pypi.io`?

Reply:
pypi.io has been deprecated for years, pypi.org is the canonical one. https://github.com/pypi/warehouse/issues/2184#issuecomment-315175958